README.md

```diff
@@ -202,6 +202,11 @@ The question directives accept the following options:
   the key in the submission data. If no key is set, note that automatically added
   keys change when the order and amount of questions is modified.
 
+The `pick-any` directive has a `partial-points` option. When `partial-points` is
+set, some of the points are granted if the submission is partially correct.
```

Comment:
You should briefly explain how partial points are calculated (at least half of the checkboxes must be correctly selected etc.).